src/core/core.adapters.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @namespace Chart._adapters
+ * @since 2.8.0
+ * @private
+ */
+
+'use strict';
+
+function abstract() {
+	throw new Error(
+		'This method is not implemented: either no adapter ' +
+		'can be found or provide an incomplete integration.'
```

Comment:
"or provide an incomplete integration" sounds a bit off to me. How about "or an incomplete integration was provided"?